definitions/vanilla_biomes.json

```diff
@@ -257,25 +260,87 @@
     },
     {
       "id": 37,
-      "name": "Mesa",
+      "name": "Badlands",
       "color": "#d94515",
       "temperature": 2.0,
       "humidity": 0.0
     },
     {
       "id": 38,
-      "name": "Mesa Plateau F",
+      "name": "Wooded Badlands Plateau",
       "color": "#b09765",
       "temperature": 2.0,
       "humidity": 0.0
     },
     {
       "id": 39,
-      "name": "Mesa Plateau",
+      "name": "Badlands Plateau",
       "color": "#ca8c65",
       "temperature": 2.0,
       "humidity": 0.0
     },
+    {
+      "id": 40,
+      "name": "Small End Islands",
+      "color": "#282898"
```

Comment:
I found some of the new biome temperatures (if they're to be believed) here: https://minecraft.gamepedia.com/Biome#Medium.2FLush_biomes

Reply:
I will wait until MCP is released for 1.13 and look into the sources...